packages/app/client/src/ui/pages/Account/components/Dashboard.js

```diff
@@ -0,0 +1,222 @@
+            <div className={classes.boxes}>
+              <div style={{ textAlign: 'center' }}>
+                <div className={classes.bordered}>
+                  <div className={classes.topText}>
+                    <img style={{ marginRight: '5%' }} src={Like} />
+                    <div>Likes</div>
+                  </div>
+                  <div style={{ borderBottom: '2.2px solid #15B54A' }} />
+                  <div className={classes.count}>23</div>
+                </div>
+                <Button className={classes.buttons} color="primary">
+                  Redeem likes
+                </Button>
+              </div>
+              <div style={{ textAlign: 'center' }}>
+                <div>
+                  <div className={classes.bordered}>
+                    <div className={classes.topText}>
+                      <img style={{ marginRight: '5%' }} src={Badge} />
+                      <div>Badges</div>
+                    </div>
+                    <div style={{ borderBottom: '2.2px solid #15B54A' }} />
+                    <div className={classes.count}>
+                      {props.badges &&
+                        Object.keys(props.badges).filter(
+                          key => props.badges[key]
+                        ).length}
+                      /{Object.keys(props.badges).length}
+                    </div>
+                  </div>
+                </div>
+                <Button
+                  onClick={e => handleChange(e, 6)}
+                  className={classes.buttons}
+                  color="primary"
+                >
+                  View Badges
+                </Button>
+              </div>
+            </div>
+          </BorderedInfo>
+        </Grid>
+        <Grid item xs={12} sm={12} md={6}>
+          <BorderedInfo size="normal">
+            <div className={classes.header}>
+              <Typography variant="subtitle1">MY LATEST PROJECT</Typography>
+              <Button
+                className={classes.seeAllButton}
+                onClick={e => handleChange(e, 4)}
+              >
+                see all >
+              </Button>
+            </div>
+            <div>
+              <div style={{ width: '100%', height: '150px' }}>
+                <StructuredCard
+                  author={'Victor'}
+                  date="4/26/2021 2:28"
+                  title={'best title for project'}
+                  content={
+                    'this is the best content for project in the world and its sounds great'
+                  }
+                />
+              </div>
+              <div style={{ margin: '5%' }}>
+                <Button
+                  className={classes.editButton}
+                  color="primary"
+                  variant="outlined"
+                >
+                  <img
+                    style={{ width: '14px', marginRight: '15%' }}
```

Comment:
There is a component for Images in the atoms folder, verify If it fits for this implementation purposes and feel free to improve it if you need